tests/data_files/constant/bitstring-in-dn.pem

```diff
@@ -0,0 +1,51 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
As of f0e00f14fab21a00dfc077e3206d2ff4a8ba8655, I see that the copied files are exactly those that are used as an input to `x509_cert_info` or `mbedtls_x509_crl_info`. Why these functions but not x509parse_crt_file`, `x509_parse_san`, etc.? Those functions don't care about expired certificates either.

Reply:
I miss that. I do not get failure from those tests. I will update it.

Reply:
Addressed in #7652